google/ggrc-core
# GGRC-6183 Fix cycle tasks notifications generation job

# Issue description

There was an issue in CycleTasks notifications generation job for show_daily_digest case (notifications retrieving logic was incorrect).

# Steps to test the changes

To test changes locally:
1) Reset db;
2) Create and activate wf with CycleTasks;
3) Change state of CycleTasks and generate notifications with $.post("/admin/generate_wf_tasks_notifications"). Check the result in the db and via '_notifications/show_daily_digest' endpoint.

# Sanity checklist




# PR Review checklist



# TODO

- [ ] Cover case when CycleTask was updated the day before generate_wf_tasks_notifications is called with tests.
